articles/cloud-storage/cs-how-install-cifs-ecs.md

```diff
@@ -0,0 +1,125 @@
+---
+title: How to install CIFS-ECS | UKCloud Ltd
+description: Installation Guide for CIFS-ECS Tool
+services: cloud-storage
+author: Dan Baker
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to install CIFS-ECS
+toc_fullpath: Users/FAQs/cs-how-install-cifs-ecs.md
+toc_mdlink: cs-how-install-cifs-ecs.md
+---
+
+# How to install CIFS-ECS
+
+## What is CIFS-ECS?
+
+CIFS-ECS is a free application that runs on Windows and Windows Server, exposing UKCloud Cloud Storage as file systems. It’s designed as an easy way to use object storage by enabling Windows applications to interface with Cloud Storage servers using the popular S3 REST API.
+
+If you require a fully-integrated cloud gateway appliance with features like advanced deduplication and location caching, you may want to consider [CloudArrary](cs-how-install-cloudarray.md) as an alternative. 
+
+    > [!NOTE]
+    > CloudArray is not a free application like CIFS-ECS.
```

Comment:
To make it a bit clearer, change to:
Unlike CIFS-ECS, CloudArray is not a free application.